virt/virt-4-8-release-notes.adoc

```diff
@@ -77,15 +75,17 @@ The SVVP Certification applies to:
 * Cloning a data volume into a different namespace is now faster and more efficient when using storage that supports Container Storage Interface (CSI) snapshots. The Containerized Data Importer (CDI) uses CSI snapshots, when they are available, to improve performance when you create a virtual machine from a template.
 
 //CNV-12269 When a VM frees space on a virtual disk the discard requests are passed to the underlying storage device which may free up real storage capacity.
+* When a virtual machine frees space on a virtual disk using the `fstrim` or `blkdiscard` commands, the discard requests are passed to the underlying storage device, possibly freeing up real storage capacity, provided the storage provider supports this feature.
```

Comment:
This is rather anthropomorphic. :-)  

I read the epic but it is not quite accurate to say that the VM "uses" the commands. According to [oVirt docs](https://www.ovirt.org/develop/release-management/features/storage/pass-discard-from-guest-to-underlying-storage.html),  fstrim is run manually while blkdiscard is set in the LVM. This is one of those exceedingly rare cases where I think it would be best to use the passive voice.

Suggested rewrite: 

When the `fstrim` or `blkdiscard` commands are run on a virtual disk, the discard requests are passed to the underlying storage device. If the storage provider supports the `pass discard` feature, the discard requests free up storage capacity.